addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -245,6 +245,22 @@ body.editor_enable.editor_has_snippets {
             text-transform: capitalize;
         }
 
+        we-imagepicker {
+            we-row {
+                max-width: 100%;
+            }
+
+            .o_we_edit_image {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                & > span {
```

Comment:
If you want to follow my convention and make me happy, you can remove the `&` and add a blank like after align-items: center.